authentication-typescript/template/store/local-storage.ts

```diff
@@ -0,0 +1,63 @@
+import { LOCAL_USER_DATA, REMEMBER_ME_DATA } from '../constants';
+import AsyncStorage from '@react-native-community/async-storage';
```

Comment:
this import statement needs to be updated to point to the correct package (in both languages)